### ISSUE TITLE
[FEAT] Changing the order in which the months appear

### ISSUE THREAD

**Issue by joaochris** [sws2apps/organized-app#3239]
### Feature description:

It seems strange to us here that the months appear like this. It would be good if there was an option to change the order in which the months appear. January, February

![Screenshot_2024-12-16-11-51-44-389_com android chrome](https://github.com/user-attachments/assets/00762017-c3d1-4977-9f40-8e0afbd57237)

**Comment by ux-git** [comment]
I agree that the list isn’t very intuitive. Unless you know that new months are at the top and the previous ones are at the bottom, it can be confusing. But this isn’t really intuitive, it’s just explained that way.

The problem is that both the top-to-bottom and bottom-to-top approaches make sense. You wouldn’t want the new month at the bottom, but at the same time, we all read from top to bottom. Both options are logical, which makes it tricky! 😅

But the solution might be to allow users to decide which approach works better for them, with a tiny Sort order button at the top right corner:

<img width="446" alt="Bildschirmfoto 2024-12-16 um 17 33 34" src="https://github.com/user-attachments/assets/75cd70ed-351f-4e94-ad7b-7a371d597409" />
